jenkins/ci.suse.de/ccp.yaml

```diff
@@ -0,0 +1,66 @@
+- job:
+    name: 'ccp-full-periodic'
+    node: cloud-trackupstream
```

Comment:
A note on using this particular Jenkins agent node: it only has one worker enabled, which acts as a bottleneck. It is also heavily used for important build-service related tasks by other jobs (trackupstream, submit). If you expect your job to take a long time to complete, I would really advise you to use another node or spin up one yourself.